daiquiri/datalink/adapter.py

```diff
@@ -210,8 +221,28 @@ def get_table_links(self, table):
         return table_links
 
 
+class QueryJobDatalinkAdapterMixin(object):
+    '''
+    Injects the query job into the context data for the daiquiri.datalinks.views.datalink view
+    '''
+
+    def get_context_data(self, request, **kwargs):
+        from daiquiri.query.models import QueryJob
+
+        context = super().get_context_data(request, **kwargs)
+
+        if 'job' in request.GET:
```

Comment:
It seems that the job paramter is not properly propagated to the datalink url.

Reply:
When querying with ?job=<job-id> the query_job is properly propagated to context so all fine. the problem really comes from the JS script of the table.